### PR TITLE
store: add option to disable volatile

### PR DESCRIPTION
**Pull request by giuseppe**
add an option to storage.options to inhibit usage of the volatile storage flag.

Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM as long as podman, crio and buildah will not throw an error if they ask for a volatile mount.

**Comment by giuseppe** [comment]
> LGTM as long as podman, crio and buildah will not throw an error if they ask for a volatile mount.

no, if it is disabled and CRI-O/Buildah/Podman sets the volatile flag then it is just ignored

**Comment by rhatdan** [comment]
@vrothberg @nalind @TomSweeneyRedHat PTAL

File: docs/containers-storage.conf.5.md [hunk 1]
```diff
@@ -87,6 +87,9 @@ The `storage.options` table supports the following options:
 **auto-userns-max-size**=65536
   Auto-userns-max-size is the maximum size for a user namespace created automatically.
 
+**disable-volatile**=true
+if disable-volatile is set then the "volatile" mount optimization is disabled for all the containers.
```
Comment (TomSweeneyRedHat) [inline docs/containers-storage.conf.5.md:91]:
```suggestion
  If disable-volatile is set, then the "volatile" mount optimization is disabled for all the containers.
```

**Comment by TomSweeneyRedHat** [comment]
Nit on the man page and it looks like it needs to be rebased.
Otherwise LGTM.

**Review by rhatdan — APPROVED** [review]
LGTM

**Review by TomSweeneyRedHat — COMMENTED** [review]
LGTM

**Comment by rhatdan** [comment]
Failure is not related to this PR.

**Pull request merged**